src/server/send/send.js

```diff
@@ -15,9 +17,18 @@ export const sendLinkByEmail = async (to, link) => {
     html: text,
     text
   }
-  await sgMail.send(msg).catch(error => {
+  return sgMail.send(msg).catch(error => {
     //Log friendly error
     log.error(error.toString())
     throw error
   })
 }
+
+export const sendLinkBySMS = async (to, link) => {
+  console.log({ conf })
+  const { plivoAuthID, plivoAuthToken, plivoPhoneNumber } = conf
+  const client = new plivo.Client(plivoAuthID, plivoAuthToken)
+  const text = `You got GD. To withdraw open: ${link}`
+
+  return client.messages.create(plivoPhoneNumber, to, text)
```

Comment:
Does client.message.create throws an error on fail? return failure? because on the api endpoint its only awaited on if error is in result it will not be handled correctly

Reply:
It throws an error. This is being handled by wrappedAsync and returned as error.